app/models/piece.rb

```diff
@@ -5,6 +5,19 @@ class Piece < ActiveRecord::Base
   validates :color, inclusion: { in: %w(black white) }
   validates :type, inclusion: { in: %w(Pawn Rook Bishop Knight King Queen) }
 
+  # Returns "valid" and updates the piece's x and
+  # y position to provided coordinates if move is valid,
+  # otherwise do nothing and return nil.
+  def move(x,y)
+    if valid_move?(x,y)
+      self.x_position = x
+      self.y_position = y
+      return "valid"
+    else
+      return nil
```

Comment:
See above note as well.

Can I make it return true and false but not put a huh operator as the method name, because I think that is a bit misleading as to the nature of the method.  Or is making it a huh operator the way to go even though its main feature would be its 'side effect'?